lib/strong_migrations/migration.rb

```diff
@@ -69,6 +69,14 @@ def method_missing(method, *args, &block)
 
     private
 
+    def safety_overridden?
+      @safe || ENV["SAFETY_ASSURED"] || should_assume_safety_due_to_version?
+    end
+
+    def should_assume_safety_due_to_version?
+      version && version <= StrongMigrations.assume_safety_prior_to_version
```

Comment:
@vergenzt @ankane In what cases is `version` nil?

I just added the strong_migrations gem to our Rails 3 app. Unfortunately, the `start_after` option doesn't seem to work. Old dangerous migrations fail despite me configuring `start_after` option https://gist.github.com/nisanth074/ea45c8cdd50d4a08b53cb97b91fc3a6b

I added a `binding.pry` statement above this line and ran `bundle exec rake db:create db:migrate` again. It seems like `version` is always `nil` for all our migrations. So, `should_assume_safety_due_to_version?` returns nil and strong_migrations raises an error.

Can you let me know in what cases is `version` nil? Is Rails 3 supported?

Thanks for your time.